src/errors/test-run/templates.js

```diff
@@ -169,25 +169,25 @@ export default {
     `),
 
     [TYPE.actionElementNotTextAreaError]: err => markup(err, `
-        The action element is expected to be a textarea.
+        The action element is expected to be a &lt;textarea&gt;.
 
         ${err.getCallsiteMarkup()}
     `),
 
     [TYPE.actionElementNotIframeError]: err => markup(err, `
-        The action element is expected to be an iframe.
+        The action element is expected to be an &lt;iframe&gt.
 
         ${err.getCallsiteMarkup()}
     `),
 
     [TYPE.actionIncorrectKeysError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
+        The "${err.argumentName}" argument contains an incorrect key or key combination.
 
         ${err.getCallsiteMarkup()}
     `),
 
     [TYPE.actionUnsupportedDeviceTypeError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument specifies an unsupported <code>${err.actualValue}</code> device. For a list of supported devices, refer to <a href="http://viewportsizes.com">http://viewportsizes.com</a>
+        The "${err.argumentName}" argument specifies an unsupported "${err.actualValue}" device. For a list of supported devices, refer to <a href="http://viewportsizes.com">http://viewportsizes.com</a>.
```

Comment:
`...unsupported "${err.actualValue}" device....`
In all other places we don't use "..." for `actualValue`